doc/guides/west/release-notes.rst

```diff
@@ -1,6 +1,91 @@
 West Release Notes
 ##################
 
+v0.6.x
+******
+
+- No separate bootstrapper
+
+  In west v0.5.x, the program was split into two components, a bootstrapper and
+  a per-installation clone. The bootstrapper was a program that implemented the
+  :ref:`west init <west-multi-repo-cmds>` command, which cloned the west
+  repository itself along with its other tasks. Except for ``west init``, all
+  other west commands were implemented by code in the cloned west repository,
+  which the bootstrapper delegated to. The bootstrapper itself was distributed
+  via PyPI and installed using ``pip3``.
+
+  This is similar to how Google's Repo tool works, but was determined to be a
+  mistake for west.
+
+  From v0.6.x onwards, all of the core west commands and helper classes are
+  part of the west package distributed via PyPI. This eliminates complexity
+  and makes it possible to import west modules from anywhere in the system,
+  not just extension commands.
+- The ``selfupdate`` command still exists for backwards compatibility, but
+  now simply exits after printing an error message.
+- Manifest syntax changes
```

Comment:
Maybe mention that the changes are backwards compatible

Reply:
The syntax is compatible, but the semantics are not (project names must now be unique, which was previously not required), so I don't want to mislead people and would rather not write that here.